### PR TITLE
fix(eip/associate): add check logic to check if resource exists

### PR DESCRIPTION
**Pull request by Lance52259**
**What this PR does / why we need it**:
When the ECS instance is deleted, the EIP relationship no longer exists, and it should be marked as deleted (set ID as empty string).

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note
1. add check logic to check if resource exists.
```

## PR Checklist

* [ ] Tests added/passed.
* [ ] Documentation updated.
* [ ] Schema updated.

## Acceptance Steps Performed

```
NONE
```

### CODE REVIEW

File: huaweicloud/resource_huaweicloud_compute_eip_associate.go [hunk 1]
```diff
@@ -95,7 +95,7 @@ func resourceComputeEIPAssociateCreate(d *schema.ResourceData, meta interface{})
 	// get port id
 	portID, privateIP, err := getComputeInstancePortIDbyFixedIP(d, config, instanceID, fixedIP)
 	if err != nil {
-		return fmtp.Errorf("Error getting port id of compute instance: %s", err)
+		return common.CheckDeleted(d, err, "eip associate")
```
Comment (niuzhenguo) [inline huaweicloud/resource_huaweicloud_compute_eip_associate.go]: seems we should just raise error within  Create

Comment (Lance52259) [reply huaweicloud/resource_huaweicloud_compute_eip_associate.go]: done.

**Pull request merged**